### DISCUSSION TITLE
flinkcdc 导入 doris 时 报错 current running txns on db 11002 is 100, larger than limit 100, see more in null

### DISCUSSION THREAD

**Discussion by AlvisZhao** [apache/doris#13413] [A - General / Q&A]
已经确保了当前没有其他导入任务在执行

仅启动一个flinkcdc任务向doris中导入数据（源表中数据仅有200条），出现报错current running txns on db 11002 is 100, larger than limit 100, see more in null

在此之前使用过datax，导入数据没有问题，首次出现上述错误后就再也没有导入成功过。

**Comment by LemonLiTree** [comment]
可以调整fe配置max_running_txn_num_per_db。文档说明如下：
https://doris.apache.org/zh-CN/docs/dev/admin-manual/config/fe-config#max_running_txn_num_per_db
